packages/zilliqa-js-blockchain/src/chain.ts

```diff
@@ -401,6 +402,18 @@ export class Blockchain implements ZilliqaModule {
     return this.provider.send<string, string>(RPCMethod.GetMinimumGasPrice);
   }
 
+  /**
+   * getPendingTxn
+   * See the pending status of transaction
+   * @param tx
+   */
+  getPendingTxn(tx: string): Promise<RPCResponse<PendingTxnResult, string>> {
```

Comment:
I think `txId` is more straightforward than `tx`. Can we change it?

Reply:
Cool. Will do